examples/provider.tf

```diff
@@ -8,6 +8,6 @@ terraform {
 }
 
 provider "redpanda" {
-  client_id      = "your_client_id"
-  client_secret  = "your_client_secret"
+  client_id      = "5Oe5EWVfpG0ED4HCmEdeGD6WxWQVibvu"
+  client_secret  = "oLeyHohbptWA7bmqVbJJ9tux-pMAINu9-GM_HUMGaMqYzTBKnk_0b1XtuhlSSipU"
```

Comment:
Please do not commit keys. The provider is able to access credentials stored in environment variables as defined in the docs. You will need to work with Platform to clean up the commit history/github history and recreate this client.

Reply:
Sorry about that. I kept switching between two versions of that file locally but I missed switching them back on the commit I pushed. I've deactivated the client credentials and updated the commit.